src/tsdb.c

```diff
@@ -143,13 +143,45 @@ void FreeCompactionRule(void *value) {
 	free(rule);
 }
 
+uint64_t SeriesGetChunksSize(Series *series) {
+    uint64_t size = 0;
+    Chunk *currentChunk;
+    RedisModuleDictIter *iter = RedisModule_DictIteratorStartC(series->chunks, "^", NULL, 0);
+    while (RedisModule_DictNextC(iter, NULL, (void*)&currentChunk))
+    {
+        size += GetChunkSize(currentChunk);
+    }
+    RedisModule_DictIteratorStop(iter);
+    return size;
+}
+
 size_t SeriesMemUsage(const void *value) {
     Series *series = (Series *)value;
-    return sizeof(series) + sizeof(Chunk) * RedisModule_DictSize(series->chunks);
+
+    size_t strlen = 0;
```

Comment:
@ashtul lets not use the same symbol as the strlen function, it can be confusing.